app/Notifications/Tiers/TierUpdatedNotification.php

```
@@ -0,0 +1,35 @@
+class TierUpdatedNotification extends Notification implements ShouldQueue
+{
+    use Queueable;
+
+    public function __construct(public Tiers $tier, public array $changes) {}
+
+    public function via($notifiable): array
+    {
+        return ['database'];
+    }
+
+    public function toDatabase($notifiable): array
```

PHPMD (2.15.0)

Avoid unused parameters such as '$notifiable'. (undefined)

(UnusedFormalParameter)